unfetter-discover/unfetter-ui
# Issue 1475

Associated PR: https://github.com/unfetter-discover/stix/pull/40

Instructions
- Updated NPM install of STIX package for UI (Note, STIX 0.3.11-develop tag is updated to include the relevant changes)
- Create the following STIX object:

```
db.stix.insert({
    "_id": "x-unfetter-threat-board--a86a845d-8762-4f6d-afa0-240d1425c963",
    "stix": {
        "name": "Test Threat Board",
        "reports": [
            "report--3284a25f-114a-490c-af70-ccfc694f7f02",
            "report--45a907a9-ce4c-4586-80c9-1412aaf383e1",
            "report--5abcecd7-ce21-4990-9fda-8b9afddc3990"
        ],
        "id": "x-unfetter-threat-board--a86a845d-8762-4f6d-afa0-240d1425c963",
        "type": "x-unfetter-threat-board",
        "boundaries": {
            "targets": [
                "sample target 1"
            ],
            "malware": [
                "malware--8e461ca3-0996-4e6e-a0df-e2a5bbc51ebc",
                "malware--63c2a130-8a5b-452f-ad96-07cf0af12ffe"
            ],
            "start_date": ISODate("2018-09-12T16:23:54.479-04:00"),
            "end_date": ISODate("2018-09-12T16:23:54.479-04:00")
        },
        "modified": ISODate("2018-08-31T09:38:25.824-04:00"),
        "created": ISODate("2018-08-31T09:38:10.000-04:00")
    }
})
```
- Navigate to `/threat-beta/x-unfetter-threat-board--a86a845d-8762-4f6d-afa0-240d1425c963/article` and confirm the reports are selectable
- Navigate to other threat boards in the master list dialog
- Confirm `/threat-beta/fakeid/article` takes you to an error page

fixes unfetter-discover/unfetter#1475
